.circleci/config.yml

```diff
@@ -811,7 +811,7 @@ jobs:
 
   t_ems_solcjs:
     docker:
-      - image: buildpack-deps:latest
+      - image: << pipeline.parameters.ubuntu-2004-docker-image >>
```

Comment:
I found it easier to do this than to install z3 manually here. Any objections?

Reply:
Actually still doesn't work since our libz3-static-dev doesn't ship the binary needed by the solc-js tests